tableauserverclient/server/request_factory.py

```diff
@@ -484,10 +562,102 @@ def create_req(self, site_item):
             site_element.attrib['storageQuota'] = str(site_item.storage_quota)
         if site_item.disable_subscriptions is not None:
             site_element.attrib['disableSubscriptions'] = str(site_item.disable_subscriptions).lower()
+        if site_item.subscribe_others_enabled is not None:
+            site_element.attrib['subscribeOthersEnabled'] = str(site_item.subscribe_others_enabled).lower()
+        if site_item.revision_limit:
+            site_element.attrib['revisionLimit'] = str(site_item.revision_limit)
+        if site_item.subscribe_others_enabled is not None:
+            site_element.attrib['revisionHistoryEnabled'] = str(site_item.revision_history_enabled).lower()
+        if site_item.data_acceleration_mode is not None:
+            site_element.attrib['dataAccelerationMode'] = str(site_item.data_acceleration_mode).lower()
         if site_item.flows_enabled is not None:
             site_element.attrib['flowsEnabled'] = str(site_item.flows_enabled).lower()
         if site_item.cataloging_enabled is not None:
             site_element.attrib['catalogingEnabled'] = str(site_item.cataloging_enabled).lower()
+        if site_item.editing_flows_enabled is not None:
+            site_element.attrib['editingFlowsEnabled'] = str(site_item.editing_flows_enabled).lower()
+        if site_item.scheduling_flows_enabled is not None:
+            site_element.attrib['schedulingFlowsEnabled'] = str(site_item.scheduling_flows_enabled).lower()
+        if site_item.allow_subscription_attachments is not None:
+            site_element.attrib['allowSubscriptionAttachments'] = str(site_item.allow_subscription_attachments).lower()
+        if site_item.guest_access_enabled is not None:
+            site_element.attrib['guestAccessEnabled'] = str(site_item.guest_access_enabled).lower()
+        if site_item.cache_warmup_enabled is not None:
+            site_element.attrib['cacheWarmupEnabled'] = str(site_item.cache_warmup_enabled).lower()
+        if site_item.commenting_enabled is not None:
+            site_element.attrib['commentingEnabled'] = str(site_item.commenting_enabled).lower()
+        if site_item.revision_history_enabled is not None:
```

Comment:
subscribeOthersEnabled, revisionHistoryEnabled, revisionLimit, and userQuota all repeated here as well.